docs/cloud/integrations/deepnote.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect to Steampipe Cloud from Deepnote
+sidebar_label: Deepnote
+---
+
+##  Connect to Steampipe Cloud from Deepnote
+
+[Deepnote](https://deepnote.com/home) is a collaborative notebook for discovering, understanding and sharing data that connects to many databases, including Postgres, and enables users to explore, query, and visualize data.
+
+Steampipe provides a single interface to all your cloud, code, logs and more. Because it's built on Postgres, Steampipe provides an endpoint that any Postgres-compatible client -- including Deepnote -- can connect to.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect Deepnote to Steampipe Cloud.
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+##  Connect to Steampipe CLI from Deepnote
+
+You can also connect Deepnote to [Steampipe CLI](https://steampipe.io/downloads). To do that, run `steampipe service start --show-password` and use the displayed connection details.
+
+```
+Steampipe service is running:
+
+Database:
+
+  Host(s):            localhost, 127.0.0.1, 192.168.29.204
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+[Deepnote](https://deepnote.com/sign-up) is accessible through any web-browser. Here we will create a Steampipe Cloud connection from Deepnote.
+
+To create a new connection, first click on `Projects` to create a new project. Then click `Create new` from the `Integrations` tab at the right-hand sidebar, select `PostgreSQL` and enter the connection details.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
```

Comment:
That note about whitelisting threw me off when I saw it, and on my first try I came away thinking I wouldn't be able to connect SPC to Deepnote because of that. 

Apparently it's not a blocker? Do we know why not? Should we say something like : Don't worry about the whitelist, despite how it appears it will work anyway?

Reply:
for whitelisting `Authorizing Deepnote's IP addresses is only available as a part of the [Team plan](https://deepnote.com/pricing). Academic teams are also unable to whitelist specific IP addresses.`

But it is not required for the Free plan or the custom plan.  Thats why i did not mention it since we were on the free version for testing it. 

We could mention `Whitelisting` but we would have to also mention the Team plan for that.

Reply:
I think it's worth spelling it out, I actually failed to complete my planned test because I had connection trouble that I attributed (incorrectly, I now see) to the whitelisting.

Reply:
i've updated the whitelisting note with the link to the IP addresses. Does that look good ?

Reply:
I found a link that might help with the connection issue you faced. https://community.deepnote.com/c/ask-anything/ip-address-for-postgresql-connection